scripts/sync_hive_to_snowflake.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+HIVE_CLASSPATH=$(hive -e "set env:CLASSPATH;" | grep env:CLASSPATH= | sed -e "s/^env:CLASSPATH=//")
+java -cp $HIVE_CLASSPATH:. com.snowflake.core.util.HiveSyncTool
```

Comment:
if https://github.com/snowflakedb/snowflake-hive-metastore-connector/pull/23 is merged first, update the package here.

Reply:
done